Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -352,6 +352,12 @@ protected function initialize()
             }
         }
 
+        foreach ($this->settings['reflection']['ignoredNamespaces'] as $namespace => $ignoreFlag) {
```

Comment:
I think we also need to adjust the `isTagIgnored()` method, which is used to build the flow internal reflection data.